content/en/data-center.md

```diff
@@ -13,11 +13,13 @@ The buildings housing data centers are equipped to maintain service even during
 
 ## Problem it addresses
 
-Instead of every business having to host their own server equipment where they are located, 
-data centers allow companies and individuals to take advantage of the 
-specialised knowledge and efficiencies of scale of data center providers. 
-This means not having to worry about managing power supply, fire suppression technology, 
-air conditioning or high speed internet connectivity, for example.
+Before datacenters became prevalent (late 1990ies to 2000s), there were mainly individual computers with specific tasks or those used by individuals to do their work.
+
+But computers have limited resources (disk, RAM, and memory). 
+That means that applications running on them have the same hard constraints. 
+The size of the application is determined by the capacity of the computer it's running on. 
```

Comment:
This sentence sounds weird. Maybe we can say "the performance..." or the "user experience..." or "the operability..."

Reply:
You mean "size," right? It does sound strange but don't think performance or user experience capture that. You simply can't build apps like Netflix without a datacenter, it's not a performance issue. You know what I mean?

Reply:
Yep, now I get it. Still, the sentence is weird ^^ ... Maybe, just say it like you explained: With such hard constraints, huge applications like Netflix simply cannot be operated, or This constraints are the limiting factor for applications at scale, i.e. Netflix, Spotify. ... I would love to see "at scale" instead of "size"

Reply:
Ok, have a look now